packages/atomic/src/components/search/atomic-facet-manager/atomic-facet-manager.tsx

```diff
@@ -100,6 +100,10 @@ export class AtomicFacetManager implements InitializableComponent {
 
     children.forEach((child) => {
       this.isPseudoFacet(child) && facets.push(child);
+
+      if (child.tagName === 'ATOMIC-AUTOMATIC-FACET-BUILDER') {
+        facets.push(child as BaseFacetElement);
+      }
     });
 
     return facets;
```

Comment:
Fix in the get facet to make sure that the automatic facets gets always pushed to the bottom of the facet manager.

Reply:
Guess it wouldn't be possible for a child to have true for both isPseudoFacet and the tagName check?

Reply:
Wonder if this would read better with a `.filter()` instead of conditional push with forEach?

Reply:
Yes it would be better. I didnt think about improving the function as a whole !